feature/mypage/src/main/java/com/susu/feature/mypage/main/MyPageDefaultScreen.kt

```diff
@@ -239,6 +242,21 @@ fun MyPageDefaultScreen(
     }
 }
 
+private fun getAppVersion(context: Context): String {
+    try {
+        val packageInfo = context.packageManager.getPackageInfo(
+            context.packageName, 0,
+        )
+
+        if (packageInfo != null) {
+            return packageInfo.versionName
+        }
+    } catch (e: Exception) {
+        e.printStackTrace()
+    }
+    return ""
+}
```

Comment:
```suggestion
private fun getAppVersion(context: Context): String = runCatching {
        val packageInfo = context.packageManager.getPackageInfo(
            context.packageName, 0,
        )

        packageInfo.versionName
}.getOrNull() ?: ""
```

모든 예외를 catch 할거면 runCatching이 더 가독성이 좋을거같아요

또 이 함수는 리컴포지션이 발생할 때마다 실행되서 별로인거같습니다. remember 쓰는거 어때요?

Reply:
좋네요, 반영했습니다! [refactor: getAppVersion 함수 runCatching으로 변경 및 remember로 호출 최소화](https://github.com/YAPP-Github/oksusu-susu-android/pull/91/commits/73f18a0a2821162c5ef1825ccb81e792a8505a9f)